pkg/stringhelper/stringhelper.go

```diff
@@ -36,6 +36,16 @@ func StringInSlice[T ~string](s []T, str T, contains bool) bool {
 	return false
 }
 
+// SubSlice checks if a slice's elements all exist within a slice
+func SubSlice[T ~string](s, sub []T) bool {
```

Comment:
Is it a good taste to use generics? It is a string helper.

Reply:
Works either way.  I can switch it just accept `[]string` variables if needed.

Reply:
I think it should be string type in this case.

Reply:
Generics removed.